src/modules/musig/tests_impl.h

```diff
@@ -170,6 +170,42 @@ void musig_api_tests(secp256k1_scratch_space *scratch) {
     CHECK(secp256k1_musig_pubkey_combine(vrfy, scratch, &combined_pk, &pre_session, pk, 2) == 1);
     CHECK(secp256k1_musig_pubkey_combine(vrfy, scratch, &combined_pk, &pre_session, pk, 2) == 1);
 
+    /** Twerking */
```

Comment:
:peach:

Reply:
Reminds me of https://github.com/ElementsProject/secp256k1-zkp/blob/fabc8f74e7e4d624d4d2d1aba621b8a75c729e5e/src/ecdsa_impl.h#L164 
 :wrench: :blue_car:  
I didn't touch this so far because it's too funny to fix.

Reply:
Was wondering if you'd find this :) Joke's over. I removed it.